lib/watcher/cpu-profile.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
```

Comment:
modify lib/watcher.js to ignore watchers that don't export a function, massive changes like this will cause you to tear out your hair resolving conflicts against the other supervisor branches